app/views/layouts/mailer.html.erb

```diff
@@ -98,7 +98,7 @@
                           <%= t 'mailer.general.footer.general.html', support_url: root_url + "support", donate_url: donate_url %>
 
                           <% if content_for?(:sent_at) %>
-                            <p><%= t 'mailer.general.footer.sent_at', sent_at: :sent_at %></p>
+                            <p><%= t 'mailer.general.footer.sent_at', sent_at: yield(:sent_at).strip %></p>
```

Comment:
The `.strip` was necessary because otherwise it includes any spaces used for indenting, e.g.

```
<% content_for :sent_at do %>
  <%= Time.now.to_s(:time_for_mailers) %>
<% end %>
```

I'd have included a nice little comment in the code if the mailer views weren't so darn sensitive about the spacing. :|